src/i18n.js

```diff
@@ -0,0 +1,19 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import Backend from "i18next-xhr-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    fallbackLng: ["en"],
+    whitelist: ["en", "sv", "cat"],
+    debug: true,
```

Comment:
you can remove the `debug: true`, it's not needed (sorry for messy demo)

Reply:
fixed. it's removed :)